manifests/init.pp

```diff
@@ -31,34 +31,35 @@
 #
 #
 class firewalld (
-  Enum['present','absent','latest','installed'] $package_ensure = 'installed',
-  String $package = 'firewalld',
-  Stdlib::Ensure::Service $service_ensure = 'running',
-  String  $config_package            = 'firewall-config',
-  Boolean $install_gui               = false,
-  Boolean $service_enable            = true,
-  Hash    $zones                     = {},
-  Hash    $ports                     = {},
-  Hash    $services                  = {},
-  Hash    $rich_rules                = {},
-  Hash    $custom_services           = {},
-  Hash    $ipsets                    = {},
-  Hash    $direct_rules              = {},
-  Hash    $direct_chains             = {},
-  Hash    $direct_passthroughs       = {},
-  Boolean $purge_direct_rules        = false,
-  Boolean $purge_direct_chains       = false,
-  Boolean $purge_direct_passthroughs = false,
-  Boolean $purge_unknown_ipsets      = false,
-  Optional[String] $default_zone     = undef,
-  Optional[Enum['off','all','unicast','broadcast','multicast']] $log_denied = undef,
-  Optional[Enum['yes', 'no']] $cleanup_on_exit = undef,
-  Optional[Integer] $minimal_mark = undef,
-  Optional[Enum['yes', 'no']] $lockdown = undef,
-  Optional[Enum['yes', 'no']] $ipv6_rpfilter = undef,
-  Optional[String] $default_service_zone  = undef,
-  Optional[String] $default_port_zone     = undef,
-  Optional[String] $default_port_protocol = undef,
+  Enum['present','absent','latest','installed']                 $package_ensure            = 'installed',
```

Comment:
Given the propensity for data types to be quite long, aligning them can get pretty crazy. It also causes a lot of needless code churn that makes reviews harder to do.